### ISSUE TITLE
Make the Omnibox keyword configurable

### ISSUE THREAD

**Issue by sferencik** [brandon1024/find#405]
## Issue Description

Please make the Omnibox keyword configurable. "find" is a useful search term, and it seems quite a few of my Google searches (which I type in the Ominibox) start with the term "find".

## Steps to Reproduce
- Start typing your Google search query in the Chrome Omnibox (without the quotes): "find the serial number of X"
- Expected: I get a Google search with how to find the serial number of X
- Actual: After typing "find the" I notice that find+ has kicked in, and I have to start again, e.g. with "<space>find" or "g<space>find" or "<ctrl+k>find". Of course I never remember to start my Chrome searches with space, just because the first word is "find".

## Praise for find+

I love find+, thank you for it. I haven't used it much yet, but the one time I did use it, it saved me so much toil.

**Comment by pzhlkj6612** [comment]
For now, I always hit <kbd>Backspace</kbd> to exit find+ after typing "find".

**Comment by dezza** [comment]
I agree, its annoying to have `find` a keyword in the address bar without toggle.

Also totally unexpected.

**Comment by dezza** [comment]
https://github.com/brandon1024/find/pull/407

Better than no change.

**Comment by brandon1024** [comment]
Keyword changed to `findplus` in #422, slated for release in 2.2.3.

**Comment by brandon1024** [comment]
The latest release now uses the `findplus` keyword. Unfortunately, the keyword cannot be made configurable because it's defined in the extension manifest.
